rulesets/resources/301ruleset.rb

```diff
@@ -1308,6 +1308,14 @@ def self.set_systems_hvac_rated(orig_bldg, new_bldg)
                                    integrated_heating_system_efficiency_percent: orig_cooling_system.integrated_heating_system_efficiency_percent,
                                    integrated_heating_system_fraction_heat_load_served: orig_cooling_system.integrated_heating_system_fraction_heat_load_served,
                                    htg_seed_id: htg_seed_id)
+
+      next unless [HPXML::HVACTypeCentralAirConditioner, HPXML::HVACTypeMiniSplitAirConditioner].include?(orig_cooling_system.cooling_system_type) && orig_cooling_system.compressor_type == HPXML::HVACCompressorTypeVariableSpeed
```

Comment:
I don't think this is necessary, you can rely on OS-HPXML to handle the inputs appropriately.